website/docs/components/table/partials/guidelines/guidelines.md

```diff
@@ -100,36 +159,83 @@ Row placement determines the visual styling based on where the row is placed rel
+![Icon within a table without a label](/assets/components/table/icon-without-label.png)
+!!!
 
-Icons should rarely be used without a text label. A label helps reinforce the purpose and communication of the icon and can reduce ambiguity when expressing complex data.
+!!! Dont
 
-!!! Do
+Don’t use an icon to indicate the status of an object, row, or resource. Instead, use a [Badge](components/badge).
 
-![Example of proper icon usage in tables](/assets/components/table/table-nested-icons-do.png =726x*)
+![Icons being used for status](/assets/components/table/icon-status.png)
 !!!
 
-#### Leading vs. trailing icons
+#### Service icons
+
+Use service icons within a cell to communicate the source or provider of a service.
 
-Both leading and trailing icons increase the visual weight of the content within the cell, so use icons intentionally throughout Tables. Take care not to mix and match different icon positions in the same column. 
+![Service icon within a table](/assets/components/table/with-service-icon.png)
+
+#### Grouping
+
+Use icons to communicate commonalities between values or that a value is part of a larger object or hierarchical structure.
+
+![Icons used for grouping in a table](/assets/components/table/icon-grouping.png)
+
+#### Product branding
+
+Use icons to communicate that a specific item is a HashiCorp product or resource.
+
+![Icon product branding](/assets/components/table/icon-product-branding.png)
+
+!!! Dont
+
+Don’t use an [Icon Tile](/assets/components/icon-tile) in place of an icon within a table cell.
+
+![Icon Tile within a table cell](/assets/components/table/icon-tile-product-branding.png)
+!!!
+
+#### Leading vs. trailing icons
 
 In general, we recommend using **leading icons** because the text following the icon will remain aligned and thus be easier for the user to scan. 
 
-!!! Do
+Take care not to mix and match different icon positions in the same column. 
+
+### Links within cells
 
-![Example of proper icon placement in tables](/assets/components/table/table-icons-placement-do.png =784x*)
+!!! Info
+
+This guidance is an extension of the [Inline Link](/components/link/inline) guidelines.
 !!!
 
-!!! Dont
+Within a table, use `secondary` (`Foreground / Strong`) links as the default. 
+
+Use `Body / 200 / Link` as the default typographic style within a table. This style increases the prominence a small amount to differentiate it from other string and non-interactive content.
+
+![Link example](/assets/components/table/link-example.png)
 
-![Example of proper icon placement in tables](/assets/components/table/table-icons-placement-dont.png =380x*)
-!!!
\ No newline at end of file
+#### Multiple links
+
+If a table contains more than one column of links, consider using `Body / 200 / Link` for the most important link (usually the title of the row, ID, or other naming convention), and `Body / 200 / Regular` with an underline for less important links.
```

Comment:
`with an underline` - shouldn't all inline links have underlined text anyway?

Reply:
This is the differentiation between the styles in Figma, `Body / 200 / Link` in Figma has an underline as part of the style, but `Body / 200 / Regular` does not and one must be added manually. I'll clear up the language a bit here though